scripts/release/tag_images.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+"""Tags images used in a particular release and pushes the tags into
+our local docker registry at docker.montagu.dide.ic.ac.uk:5000.  Use
+version 'latest' to select the most recent conforming git tag (this will
+not set things to be the docker 'latest' tag though).  If run without the
+"--local-only" option it will also publish images to
+https://hub.docker.com/u/vimc and publish a 'release' tag there which will
+always give the latest released public version.
+
+Usage:
+  tag-images.py <version> [--local-only]
```

Comment:
So there's no longer a "publish" command? I think we need this for the case where you run a 2 step process: "tag --local-only" followed by "publish"?